jobserver/models.py

```diff
@@ -487,3 +488,37 @@ def repo_name(self):
         if not f.path:
             raise Exception("Repo URL not in expected format, appears to have no path")
         return f.path.segments[-1]
+
+
+class Release(models.Model):
+    id = models.TextField(primary_key=True)
+    published_on = models.DateTimeField(default=timezone.now)
+    upload_dir = models.TextField()
+    # list of files in the release upload
+    files = models.JSONField()
+
+    workspace = models.ForeignKey(
+        "Workspace",
+        on_delete=models.PROTECT,
+        related_name="releases",
+    )
+    backend = models.ForeignKey(
+        "Backend",
+        on_delete=models.PROTECT,
+        related_name="+",
+    )
```

Comment:
Can you move the FKs to the top of the model definition?  I find it's really useful for scanning the model definitions if they're all in one place.

(Sorry, I realise there's no way you could have known about this given there's zero docs on the format!)